### ISSUE TITLE
Sails new ./ breaks npm package file

### ISSUE THREAD

**Issue by mauriciogior** [balderdashy/sails#3735]
**Sails version**: 0.12.1
**Node version**: 4.2.1
**NPM version**: 2.14.7
**Operating system**: OSx

`sails new [path_to_new_app]` breaks npm package file when using `./` , since it uses it as the app name.

Suggestion: modify to `sails new [path_to_new_app] [name_of_new_app]`

<hr/>

**Comment by sailsbot** [comment]
Hi @mauriciogior!  It looks like you may have removed some required elements from the initial comment template, without which I can't verify that this post meets our [contribution guidelines](http://bit.ly/sails-issue-guide). To re-open this issue, please copy the template from [here](https://raw.githubusercontent.com/balderdashy/sails/master/.github/ISSUE_TEMPLATE), paste it at the beginning of your initial comment, and follow the instructions in the text. Then post a new comment (e.g. "ok, fixed!") so that I know to go back and check.

Sorry to be a hassle, but following these instructions ensures that we can help you in the best way possible and keep the Sails project running smoothly.

*If you feel this message is in error, or you want to debate the merits of my existence (sniffle), please contact inquiries@treeline.io.*

**Comment by mauriciogior** [comment]
fixed

**Comment by sailsbot** [comment]
Sorry to be a hassle, but it looks like your issue is still missing some required info.  Please double-check your initial comment and try again.

*If you feel this message is in error, or you want to debate the merits of my existence (sniffle), please contact inquiries@treeline.io.*

**Comment by mauriciogior** [comment]
fixed

**Comment by sailsbot** [comment]
@mauriciogior,@sailsbot: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by mikermcneil** [comment]
@mauriciogior Thanks for the report.  Just checked and this is also a problem when doing e.g. `sails new foo/baz` I think the immediate term solution here is to fix the name in the package.json and include it in the next patch release.

For the CLI in Sails v1, I'm open to ideas (feel free to put anything related to that in a proposal).  I think that the best bet would be for the CLI to continue to err on the side of forgiveness rather than specificity though.

**Comment by mikermcneil** [comment]
@mauriciogior OK, just dug into this:

To customize the app name, you can use `--appName`.

To make the default a bit better, I just pushed https://github.com/balderdashy/sails-generate-new/commit/62eda1a7d5ac2cb6aeb10f6bd4af13bd14f6a834

If anyone reading this has time to [update the relevant docs](https://github.com/balderdashy/sails-docs/edit/master/reference/cli/sailsnew.md), that'd be awesome.

Thank you!

**Comment by Leestex** [comment]
@mikermcneil here is a [PR](https://github.com/balderdashy/sails-docs/pull/737) to update the relevant docs.

**Comment by mikermcneil** [comment]
@Leestex thanks!

**Comment by mikermcneil** [comment]
ok this is published via [sails-generate-new@0.10.28](https://github.com/balderdashy/sails-generate-new/commits/master) -- going to call it good.  If anything related to this arises in the future, please let us know via a new issue and link back here.  Thanks again @Leestex and @mauriciogior!
